src/main/java/org/sigmah/client/ui/presenter/admin/models/importer/AddImportationSchemeModelsAdminPresenter.java

```diff
@@ -138,8 +139,10 @@ public void handleEvent(BaseEvent be) {
 
 				if (model instanceof ProjectModelDTO) {
 					newImportationSchemeModelProperties.put(AdminUtil.ADMIN_PROJECT_MODEL, model);
-				} else {
+				} else if (model instanceof  OrgUnitDTO) {
 					newImportationSchemeModelProperties.put(AdminUtil.ADMIN_ORG_UNIT_MODEL, model);
+				} else {

```

Comment:
can you be more precise about your expectations with an `else if (model instanceOf ContactDTO)` (or whatever suitable class) and finishing by a catchall `else` throwing an `IllegalArgumentException`?

Same issue line 200.

Reply:
Also, rename the method and make it start with a lowercase.